### PR TITLE
Various accessibility additions

### PR DESCRIPTION
**Pull request by ndarilek**

### CODE REVIEW

**Comment by hhaidar** [comment]
+1

File: locales/en.json [hunk 1]
```diff
@@ -74,5 +74,9 @@
 	"Slug": "Slug",
 	"XMPP/Jabber": "XMPP/Jabber",
 	"Private?": "Private?",
-	"Empty for public room": "Empty for public room"
+	"Empty for public room": "Empty for public room",
+  "Toggle Navigation": "Toggle Navigation",
```
Comment (hhaidar) [inline locales/en.json]: spacing

Comment (ndarilek) [inline locales/en.json]: Ah, missed that. Should be fixed now.

**Comment by hhaidar** [comment]
Thank you

**Pull request merged**